### ISSUE TITLE
机器学习项目训练时出现内存不足错误

### ISSUE THREAD

**Issue by caojing98** [caojing98/caojing98.github.io#1]
import tensorflow as tf
model = tf.keras.Sequential([
    tf.keras.layers.Dense(128, activation='relu', input_shape=(784,)),
    tf.keras.layers.Dense(10, activation='softmax')
])
model.compile(optimizer='adam', loss='sparse_categorical_crossentropy', metrics=['accuracy'])
model.fit(x_train, y_train, epochs=10)
